patches/chrome-browser-extensions-api-identity-identity_get_auth_token_function.cc.patch

```diff
@@ -106,14 +106,15 @@ index 120ed70af8385cda741cd1f34d21751c0873d0fd..6cfe5c04ac03735f235b3f3dd931a466
 +      IsInteractionAllowed(interactivity_status_for_signin_)
 +          ? WebAuthFlow::INTERACTIVE
 +          : WebAuthFlow::SILENT,
-+      WebAuthFlow::LAUNCH_WEB_AUTH_FLOW, extension()->name());
++      WebAuthFlow::LAUNCH_WEB_AUTH_FLOW);
```

Comment:
@spylogsster I'm confused here because I thought this was a change that was already in upstream so this would go away on its own, otherwise it never should have been patched this way in the first place

Reply:
This is from https://chromium-review.googlesource.com/c/chromium/src/+/2704571, it marked as work in progress, but there is no new changes last 2 years.

Reply:
Oh, I didn't realize it was wip not merged (and seemingly abandoned). We need to revisit this patch then because this kind of patching is only ok when it's going to go away on its own with upstream changes. Can you please create a new PR to address the patch and also we need to add tests for this if there is no upstream coverage.

Reply:
sure, created https://github.com/brave/brave-browser/issues/29096